### ISSUE TITLE
Map and encompassing List scroll at the same time

### ISSUE THREAD

**Issue by johankool** [mapbox/mapbox-maps-ios#2114]
## Environment

- Xcode version: 15.2
- iOS version: 17.2
- Devices affected: iPad Pro (12.9 inch)
- Maps SDK Version: 11.1.0

## Observed behavior and steps to reproduce

If I use this snippet below, both the map and the containing list scroll at the same time. This makes for a dizzying experience for the user. If I use Maps SDK Version 11.0.0 instead, the behavior is as expected: only the map scrolls if the gesture starts in the map.

```swift
 var body: some View {
    NavigationStack {
      List {
        Section {
          MapReader { proxy in
            Map(viewport: $viewport) {
              if let polygon = store.region.polygon?.polygon.polygon {
                PolygonAnnotation(polygon: polygon)
                  .fillColor(StyleColor(UIColor.blue.withAlphaComponent(0.3)))
              }
              Puck2D(bearing: .heading)
            }
          }
          .aspectRatio(1.0, contentMode: .fill)
        }
      }
    }
  }
```

## Expected behavior

The behavior as in 11.0.0 is as expected.

## Notes / preliminary analysis

Regression from 11.0.0.

## Additional links and references

Correct in 11.0.0:
https://github.com/mapbox/mapbox-maps-ios/assets/68731/b203245a-760c-45a8-a332-a894350c4cff

Incorrect in 11.1.0:
https://github.com/mapbox/mapbox-maps-ios/assets/68731/dbd35a84-4c41-4960-8898-8f93a0381957

**Comment by pjleonard37** [comment]
Hi @johankool -- thanks for reporting. We're taking a look into this now.

**Comment by aleksproger** [comment]
Hi, @johankool it's indeed a bug introduced in 11.1 and we're working to fix this
And I also want to ask what is your use case? I mean why you need to have interactable map in a scrollable container like List? 
For example, if you want to show static map with several annotations in List then you may use `MapSnapshotter` and just show snapshots of the map.
Or map is just a part of some kind of feed where other sections represent various data?

**Comment by aleksproger** [comment]
@johankool the issue was fixed and will be a part of `11.2.0-rc.1` release

**Comment by johankool** [comment]
Confirmed fixed in the 11.2.0-rc.1 release. Thanks! 👍
